src/helpers/voucherHelper.ts

```diff
@@ -55,3 +56,27 @@ export const voucherMemberId = async () => {
 
   return memberId;
 };
+
+export const storeVoucherMemberLoginInfo = (memberLoginInfo?: string) => {
+  if (memberLoginInfo) {
+    addToStore(VOUCHER_MEMBER_LOGIN_INFO, memberLoginInfo);
+  } else {
+    addToStore(VOUCHER_MEMBER_LOGIN_INFO);
+  }
+};
+
+export const voucherMemberLoginInfo = async () => {
+  let memberLoginInfo = {};
+
+  // The reason for the problem of staying in SplashScreen that occurs after the application is
+  // updated on the Android side is the inability to obtain the token here.
+  // For this reason, try/catch is used here and the problem of getting stuck in SplashScreen is solved.
```

Comment:
does the comment belong here or is it a left over copy & paste?

Reply:
Copy and paste.

Reply:
and does it belong here?